aequilibrae/paths/linear_approximation.py

```diff
@@ -426,6 +435,11 @@ def execute(self):
                 self.cores,
             )
 
+            for c in self.traffic_classes:
```

Comment:
Updates the arrays used for skimming when travel time is skimmed for any of the modes